Cargo.toml

```diff
@@ -61,6 +61,7 @@ which = "4.4.0"
 sysctl = "0.5.4"
 walkdir = "2.3.3"
 indexmap = { version = "2.0.2", features = ["serde"] }
+export = { git = "https://github.com/DeterminateSystems/export" }
```

Comment:
Before this PR can merge this git dependency either needs to be published to crates.io or pulled into this repo as a workspace.

Since `nix-installer` is published to crates.io, we cannot have git deps.